code/modules/food_and_drinks/kitchen_machinery/microwave.dm

```diff
@@ -10,6 +10,8 @@
 	active_power_usage = 100
 	circuit = /obj/item/circuitboard/machine/microwave
 	pass_flags = PASSTABLE
+	light_color = LIGHT_COLOR_YELLOW
+	light_power = 3
```

Comment:
Who's responsible for this atrocious use of light_power

Reply:
Moth sympathizers

Reply:
🔫